applications/openshift/ocp_idp_no_htpasswd/rule.yml

```diff
@@ -0,0 +1,16 @@
+prodtype: ocp4
+
+title: "The Identity Providers don't use the htpasswd-based authentication"
+
+description: TBD
+
+rationale: TBD
+
+identifiers:
+    cce@ocp4: 84209-6
+
+severity: medium
+
+warnings:
+    - general: |-
+        {{{ openshift_cluster_setting("/apis/config.openshift.io/v1/oauths/cluster", "idp.yml") | indent(8) }}}
```

Comment:
Why wasn't the path used as a file name instead of having to specify a filename with every rule?

e.g. the file could have just been `/apis/config.openshift.io/v1/oauths/cluster` instead of having to specify `idp.yml`. The parent directory creation would be handled by the operator.

Reply:
this would work, it actually might be better at preventing duplicate file names in rules

Reply:
So just a maybe stupid check whether I understood you correctly - is it like that you would be happier to save (and scan) the file to `$ROOT/apis/config.openshift.io/v1/oauths/cluster` instead to `$ROOT/idp.yml`?

Reply:
@matejak that's correct.